snapshot-labs/snapshot
# feat(lens): add lens resolver to the project

### Issues
#3853 


### Changes 
1. Added Lens Protocol SDK for resolving addresses

### How to test
1. Login with your wallet
2. go to Cultivator DAO space
3. there is only one proposal, go to it
4. check votes. You can ensure that their wallet addresses are replaced with lens handles.
5. you can also open [the version of snapshot](https://snapshot.org/#/cultivatordao.eth/proposal/0xd512345eb6c9b7bbf57b28a8b4f7dc46d31ab6c19ac8ad5de4605fa52fcb9e98) that is without lens resolver. And see the difference between lists

### To-Do
- [ ] find a way how to test it


### Self-review checklist
